### ISSUE TITLE
Optional "Favorites" Folder (Contains shortcuts to favorited folders)

### ISSUE THREAD

**Issue by Reiko9** [Frooxius/NeosPublic#555]
Simply Select a Folder in Inventory, then click the heart button to add it into the root of your favorites folder... (Folder used for Favorites can be changed so eg "Halo Favorites" can be next place that hearted folders get "copied" into)

This would be a Fast and simple method of quickly accessing content buried Deep within the hierarchy of Inventory and be Very easy to explain to new-users...
    (If a User does Not want this folder, it can be hidden with a Check-mark within settings)

A method of directly modifying folder-hierarchy exists, but it takes more effort for basic usage like I am describing.
   "ProbablePrimeToday at 2:45 PM
You can turn any folder into a spawnable item with the envelope icon, using that you can re-arrange your inventory of other public folders by just re-saving public folders you have.

Personally I Love this method posted by ProbablePrime due to its flexibility...
(Just its a bit complex to learn at first)

Select folder Item ==> Spawn with Envelope Icon ==> Grab item with laser ==> click green ( + ) with Opposite hand Laser, while at the same time still Holding down the Grip with the Item folder holding hand... (Also ensure the correct directory is selected in inventory view)

Vs
__________________________________________________________________________
Select Folder item ==> Click "Heart Button"   Done... Shortcut to folder now exists in Favorites Folder
__________________________________________________________________________
Anyways obtaining irreducible complexity, while still maintaining maximum flexibility is not usually possible, this is why it is said the User-interface will have a surface layer simple enough for end-users, but under the surface controls flexible enough for Power-users...
     It may be wise to have an "End-user" or "Power-user" tag on some Github submissions to help reduce fears that an End-user feature is going to ruin/replace a power-user feature...

**Comment by shiftyscales** [comment]
Once the inventory is overhauled during the UI update, there are a number of ways we could improve general asset browsing. Other neat potential ideas would be recently used/frequently used sorting for the inventory, separating out files by asset type, etc.

Similarly, moving items, etc. without resaving/using public folders are other quality of life features that could be integrated as a part of the overhaul.

**Comment by Reiko9** [comment]
While we are at it... Metadata on Items so Inventory can be searched by "Class/type" of device...
Aka "weapons" tag in search shows all Items tagged with "weapon" metadata... (same idea as any basic search engine on artwork websites)

**Comment by Frooxius** [comment]
Yes! I don't plan on modifying the current inventory, but favorites and shortcuts are planned for the new one! The inventory also already stores a bunch of metadata about the items, which will allow for things like search and filtering, so that will be utilized as well.

**Comment by epicEaston197** [comment]
could we also favorite avatars and items into this folder as well?
and this would not take up any space because it would just take it from the public folder that it's from right?
